src/main/java/core/basesyntax/separator/CreateTransactionsImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.separator;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CreateTransactionsImpl implements CreateTransactions {
+    private static final String SEPARATOR = ",";
+    private static final int TITLE_LINE = 1;
+    private static final int TYPE_OPERATION_INDEX = 0;
+    private static final int FRUIT_NAME = 1;
+    private static final int FRUIT_QUANTITY = 2;
+
+    @Override
+    public List<Transaction> getAllTransactions(List<String> readLine) {
+        return readLine.stream()
+                .skip(TITLE_LINE)
+                .map(s -> s.split(SEPARATOR))
+                .map(s -> new Transaction(new Fruit(s[FRUIT_NAME],
+                        Integer.parseInt(s[FRUIT_QUANTITY])),
+                        Transaction.Type.getTypeOperation(s[TYPE_OPERATION_INDEX])))
```

Comment:
I think we could move logic of creating new Transaction in private method and call it from stream (it will be more readable)